### DISCUSSION TITLE
disable zsh-autocomplete when i don't type anything

### DISCUSSION THREAD

**Discussion by MohammadRostamiorg** [marlonrichert/zsh-autocomplete#402] [Q&A] [answered]
hello 

I want to disable autocomplete when I don't type anything

look at the picture
![Screenshot_2022-01-17_15-49-27](https://user-images.githubusercontent.com/80725665/149768449-5ae99b20-072e-4978-b532-6457b281429f.png)
I don't like showing the folders name now!

**Comment by marlonrichert** [comment, marked as answer]
```zsh
zstyle ':autocomplete:*' min-input 1
```
As documented here:
https://github.com/marlonrichert/zsh-autocomplete/blob/39423112977a8c520962bc11c46ee31e7ca873ca/.zshrc#L15
